test/infrastructure/docker/config/rbac/dockerclustertemplate_editor_role.yaml

```diff
@@ -0,0 +1,24 @@
+# permissions for end users to edit dockerclustertemplates.
```

Comment:
We usually don't have the editor_role file for other objects (they were not generated by to the version of kubebuilder originally used).
I suggest to remove those files given that we are not loading them in the kustomize.yaml file.

RBAC role will be eventually generated when modifying // +kubebuilder:rbac in the controllers